pkg/instancegroups/rollingupdate_test.go

```diff
@@ -538,7 +538,11 @@ func (v *flappingClusterValidator) Validate() (*validation.ValidationCluster, er
 func TestRollingUpdateFlappingValidation(t *testing.T) {
 	c, cloud, cluster := getTestSetup()
 
-	c.ValidationTimeout = 20 * time.Millisecond
+	// This should only take a few milliseconds,
+	// but we have to pad to allow for random delays (e.g. GC)
+	// TODO: Replace with a virtual clock?
+	c.ValidationTimeout = 200 * time.Second
```

Comment:
200 seconds is a  lot of time. I'm sure 1 second would be more than sufficient.

Reply:
Oops... guess who switched from "200ms is enough" to thinking "hmm, that looks oddly specify; better just to call it one second" :-)